components/toolbar/style.scss

```diff
@@ -47,11 +59,16 @@
 		font-size: 10px;
 		font-weight: bold;
 		position: relative;
-		top: -1px;
-		margin-left: -3px;
+		top: -5px;
+		left: -9px;
 	}
 }
 
+// compensate for making the button hit area include the visual spacing
+.components-toolbar__control.components-button + .components-toolbar__control.components-button {
+	margin-left: -3px;
```

Comment:
Yeah, so this one is a bit hard to explain, to be honest, but it seems to be visually working for me, so if you're seeing any visible weirdness do let me know.

The thing is —  the toolbar _background_, the wrapping container, is now 38px tall including the borders. 

Inside this, are 36x36px buttons. So that means they take up the FULL vertical space. 

Now these 36x36px buttons have 3px padding. This adds that 3px white bit of space around the dark gray toggled version. This was previously margin.

The thing is, if you put two of these 36x36px buttons next to each other, the white space between them will be 6px. By adding a negative 3px margin to `element + element` we essentially ensure that that spacing between button is always 3px, when there's more than one.